src/include/lsm.h

```diff
@@ -162,6 +162,9 @@ struct __wt_lsm_manager {
 #define	WT_LSM_MAX_WORKERS	20
 #define	WT_LSM_MIN_WORKERS	3
 	WT_LSM_WORKER_ARGS lsm_worker_cookies[WT_LSM_MAX_WORKERS];
+
+#define	WT_LSM_MANAGER_SHUTDOWN	0x01	/* Manager has shut down */
```

Comment:
I don't love introducing a special shutdown path for LSM worker threads. The path I'd like to take this down is switching LSM over to using the `thread_group` code that the eviction server currently uses. I thought there was a JIRA ticket describing that change, but I can't find it. There is outstanding work in WT-3123 that changes the (internal) thread group API, so I'm inclined to hold off until that's resolved.